Scripts/Interfaces/IUsesSpatialHash.cs

```diff
@@ -1,18 +1,23 @@
-﻿using System;
+﻿#if UNITY_EDITOR
+using System;
 using UnityEngine;
 
-/// <summary>
-/// Make use of the spatial hash
-/// </summary>
-public interface IUsesSpatialHash
+namespace UnityEditor.Experimental.EditorVR
 {
 	/// <summary>
-	/// Add all renderers of a GameObject (and its children) to the spatial hash for queries, direct selection, etc.
+	/// Make use of the spatial hash
```

Comment:
not directly related to this PR but I think it worth mentioning: 

can we come up with better names for interfaces named like  **IUses**_Whatever_ ?

Maybe **ISupport**_Whatever_ ?